### PR TITLE
Activate BlackDuck security scan

### PR DESCRIPTION
**Pull request by dasormeter**
Activate BlackDuck security scan.

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -37,3 +37,8 @@ steps:
     versionSpec: $(nodejs_version)
 - script: npm run ci-pipeline
   displayName: Build and test
+- bash: bash <(curl -s https://raw.githubusercontent.com/DACH-NY/security-blackduck/master/synopsys-detect) ci-build digitalasset_ex-daml-js master --logging.level.com.synopsys.integration=DEBUG
+    displayName: 'Blackduck Scan'
```
Comment (stefanobaghino-da) [inline azure-pipelines.yml]: Build is failing, probably it's this indentation? Spaces are meaningful in YAML, right?

**Comment by stefanobaghino-da** [comment]
@dasormeter For visibility, would you mind opening a ticket and adding a description, please? Thanks!

**Comment by stefanobaghino-da** [comment]
@dasormeter I tried fixing the indentation problem for you but apparently there is still an issue with a missing environment variable. Please have a look at my latest two commits ([`0ec78ff`](https://github.com/digital-asset/daml-js/pull/48/commits/0ec78ffe551c63e79b48a3108446708f799ff370), [`a547113`](https://github.com/digital-asset/daml-js/pull/48/commits/a547113c303da37f08e7c7875b96166187f50a84)) to see how I tried to fix this. The first one fixed what is the expected name from the BlackDuck application itself and I expect it to be right (build output [before](https://dev.azure.com/digitalasset/daml/_build/results?buildId=6150) and [after](https://dev.azure.com/digitalasset/daml/_build/results?buildId=6151)). The second was more of a goofy attempt and failed. I believe you'd be the best person to fix this. Please note that this build does not (and unfortunately cannot†) use the DA Azure Pipelines build agents pool machines, in case you expect that environment variable to be there as it is set up for DA build agents.

† reason is since the integration tests have been introduced, `java` must be installed on the build agent, but since this repo has no `dev-env` we fell back on the Azure Pipelines native build agents pool rather than DA's.

**Comment by stefanobaghino-da** [comment]
@dasormeter Your attention is required to move this PR forward. Please have a look at the [previous comment](https://github.com/digital-asset/daml-js/pull/48#issuecomment-494286674).

**Comment by dasormeter** [comment]
sure @stefanobaghino-da will have a look

**Comment by dasormeter** [comment]
hiya @stefanobaghino-da sorry i did a naughty force merge that lost the history but thinkt he problem was naming and accessibility of the token variable + the fact that bash needs to be invoked as a separate process.  New candidate for merge to take this place is #66

**Comment by stefanobaghino-da** [comment]
Thanks for opening the issue. I used the latest commit reported in this PR to restore the work done so far in #67. As made evident by #66, the issue was simply a typo in the project name. Applied to #67 and closed #66. Too bad the original branch got deleted here on remote, otherwise we could have simply kept this PR. :wink:

**Pull request closed without merging**